src/libs/OptionsListUtils.js

```diff
@@ -150,11 +150,18 @@ function addSMSDomainIfPhoneNumber(login) {
  *
  * @param {Array<Number>} accountIDs
  * @param {Object} personalDetails
+ * @param {Object} defaultValues {login: accountID} In workspace invite page, when new user is added we pass available data to opt in
  * @returns {Object}
  */
-function getAvatarsForAccountIDs(accountIDs, personalDetails) {
+function getAvatarsForAccountIDs(accountIDs, personalDetails, defaultValues = {}) {
+    const reversedDefaultValues = {};
+    _.map(Object.entries(defaultValues), (item) => {
+        reversedDefaultValues[item[1]] = item[0];
```

Comment:
I'm slightly confused - `item` is a two-value array? Where `item[0]` is a login and `item[1]` is an accountID?

This seems very specific to workspace stuff where we use "*emailsToAccountIDs*" stuff... Even the function comment agrees it's only for workspace stuff... I feel like this would have been better as a separate function or something 🤷  this all looks terribly confusing for the standard use-case 😅 

@mountiny sorry to jump in to a PR you reviewed, do you agree or disagree with my annoying thoughts? 😅